pipe.go

```diff
@@ -145,6 +163,13 @@ func DialPipe(path string, timeout *time.Duration) (net.Conn, error) {
 	var h syscall.Handle
 	for {
 		h, err = createFile(path, syscall.GENERIC_READ|syscall.GENERIC_WRITE, 0, nil, syscall.OPEN_EXISTING, syscall.FILE_FLAG_OVERLAPPED|cSECURITY_SQOS_PRESENT|cSECURITY_ANONYMOUS, 0)
+		if err == nil {
+			break
+		}
+		if shouldRetry(err) {
```

Comment:
This looks to me like it creates an infinite loop if there really is no pipe server present.  That's bad.

There needs to be something to catch this.  Probably what you could do is move this check to after the timekeeping validation.